<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-02T10:45:41 COT" name="Suite" finished-at="2022-03-02T10:45:54 COT" duration-ms="13456">
    <groups>
    </groups>
    <test started-at="2022-03-02T10:45:41 COT" name="Test" finished-at="2022-03-02T10:45:54 COT" duration-ms="13456">
      <class name="com.testNG.RunTestNG">
        <test-method is-config="true" signature="beforeClass(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testNG.RunTestNG@6d2a209c]" started-at="2022-03-02T10:45:41 COT" name="beforeClass" finished-at="2022-03-02T10:45:43 COT" duration-ms="1868" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[https://demo.guru99.com/selenium/newtours/index.php]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[./imagenes/logo_periferia.png]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[./output/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="pruebaInicialTestNG(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testNG.RunTestNG@6d2a209c]" started-at="2022-03-02T10:45:44 COT" name="pruebaInicialTestNG" description="acceso a periferia" data-provider="busqueda" finished-at="2022-03-02T10:45:54 COT" duration-ms="10579" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Si]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Si]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[pepito]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at com.testNG.RunTestNG.pruebaInicialTestNG(RunTestNG.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pruebaInicialTestNG -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:com.testNG.RunTestNG@6d2a209c]" started-at="2022-03-02T10:45:54 COT" name="afterClass" finished-at="2022-03-02T10:45:54 COT" duration-ms="127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.testNG.RunTestNG -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
